test/tests/performance/pbs_jobperf.py

```diff
@@ -153,7 +153,7 @@ def test_job_performance_sched_off(self):
             i = 0
             users = [TEST_USER1, TEST_USER2, TEST_USER3, TEST_USER4,
                      TEST_USER5, TEST_USER6, TEST_USER7, TEST_USER,
-                     ROOT_USER, ADMIN_USER]
+                     TST_USR, ADMIN_USER]
```

Comment:
If ROOT_USER was a problem, is ADMIN_USER ok?

Reply:
It works fine, But changed it as it may cause issue in a non linux platform